backend/api/routes/users.py

```
@@ -0,0 +1,785 @@
+# =============================================================================
+# Beta Tester Program
+# =============================================================================
+
+BETA_TESTER_FREE_CREDITS = 1  # Number of free credits for beta testers
+
+
+@router.post("/beta/enroll", response_model=BetaTesterEnrollResponse)
+async def enroll_beta_tester(
+    request: BetaTesterEnrollRequest,
+    http_request: Request,
+    user_service: UserService = Depends(get_user_service),
```

Ruff (0.14.10)

Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)